### PR TITLE
minor langunage update in home page and upgraded minor version

### PR DESCRIPTION
**Pull request by KapilDuwadi**
## Checklist

* [x] The pull request title is a good summary of the changes.
* [ ] Tests for the changes exist
* [x] Tests pass on CI
* [x] Documentation reflects the changes where applicable
* [x] My PR is ready to review, **please add a comment including "please review" to assign reviewers**

### CODE REVIEW

**Review by AadilLatif — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,23 @@
 # Grid Data Models (GDM)
 
-GDM is a python package containing [pydantic](https://docs.pydantic.dev/latest/) data models for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/).
+GDM is a python package containing [pydantic](https://docs.pydantic.dev/latest/) for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/) and intended to be open sourced in the future.
```
Comment (daniel-thom) [inline README.md:3]: This sentence seems awkward. Are you intending something like this?
```suggestion
GDM is a python package containing data models for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/).
```
Note that I removed the open source language.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+
 
 ## Why Grid Data Models ?
 
 In an effort to reduce code duplication and provide client packages a standard interface to interact with power system data, a group of 
 research engineers at NREL is working on developing standard data models. Features:
 
-- **Builtin validation layer:** Use of [pydantic](https://docs.pydantic.dev/latest/) in creating data models allows us to check for fields during the time of construction and update.
-- **Timeseries data management:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) package which enables attaching time series data to fields in the data model. For example, we can attach time series power consumption data to a load profile.
+- **Builtin validation layer:** Use of [pydantic](https://docs.pydantic.dev/latest/) allows us to validate model fields.
```
Comment (daniel-thom) [inline README.md:19]: I would use `built-in`.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+- **Time series data management:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) package which enables [efficient time series data management](https://nrel.github.io/infrasys/explanation/time_series.html) by sharing arrays across components and offloading system memory. For example, we can attach time series power consumption data to a load profile.
 - **Builtin unit conversion:** GDM leverages [pint](https://pint.readthedocs.io/en/stable/) for unit conversion for power system quantities. For e.g power, voltage, time etc.
 - **JSON serialization/deserializatin:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) to serialize and deserialize distribution system containing power system components and time series data attached to components.
```
Comment (daniel-thom) [inline README.md:22]: Typo: deserialization

Comment (daniel-thom) [inline README.md:22]:
```
JSON serialization/deserialization:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) to serialize and deserialize distribution system components to/from JSON.
```
Note that we don't serialize time series data to JSON.

File: docs/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Grid Data Models (GDM)
 
-GDM is a python package containing [pydantic](https://docs.pydantic.dev/latest/) data models for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/) and intended to be open sourced in the future.
+GDM is a python package containing [pydantic](https://docs.pydantic.dev/latest/) for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/) and intended to be open sourced in the future.
```
Comment (daniel-thom) [inline docs/index.md:3]: same as above

**Review by daniel-thom — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (KapilDuwadi) [reply README.md:19]: I like that.

File: README.md (hunk 1 above)
Comment (KapilDuwadi) [reply README.md:3]: Thanks for the edit.

File: README.md (hunk 3 above)
Comment (KapilDuwadi) [reply README.md:22]: Got it.

**Pull request merged**